v3/src/components/case-table/index-menu-list.tsx

```diff
@@ -1,12 +1,44 @@
-import { MenuItem, MenuList, useToast } from "@chakra-ui/react"
-import React from "react"
+import { FormControl, FormLabel, HStack, Input, MenuItem, MenuList, Radio, RadioGroup,
+  useDisclosure, useToast } from "@chakra-ui/react"
+import React, { useRef, useState } from "react"
+import { IDataSet } from "../../data-model/data-set"
+import { CodapModal } from "../codap-modal"
 
 interface IProps {
   caseId: string
   index?: number
+  data?: IDataSet
 }
-export const IndexMenuList = ({caseId, index}: IProps) => {
+export const IndexMenuList = ({caseId, index, data}: IProps) => {
   const toast = useToast()
+  const { isOpen, onOpen, onClose } = useDisclosure()
+  const [numCasesToInsert, setNumCasesToInsert] = useState(1)
+  const [insertPosition, setInsertPosition] = useState("after")
+
+  const InsertCasesModalContent = () => {
+    const initialRef = useRef(null)
+    return (
+      <>
+        <FormControl display="flex" flexDirection="row">
+          <FormLabel># cases to insert:</FormLabel>
+          <Input size="xs" w="75" ref={initialRef} placeholder="1"
```

Comment:
When I erase the default value in the input field, NaN appears. Maybe "Number of cases" instead?